codecov.yml

```diff
@@ -0,0 +1,7 @@
+comment:
+  layout: "diff"
```

Comment:
👍 I think we don't need the heatmap (as displayed here https://github.com/boostorg/gil/pull/562#issuecomment-787508313) and just the textual info will do fine.

Reply:
Yeah this is just the textual part, you can see more details: https://docs.codecov.io/docs/pull-request-comments#layout
Additionally, we may add files too. 

![image](https://user-images.githubusercontent.com/74113200/109454359-1319f880-7a7a-11eb-979d-891479c1f7ec.png)